tests/runtime/iqm/test_iqm_runtime.py

```
@@ -0,0 +1,626 @@
+    ),
+}
+FAKE_IQM_ALIASES = tuple(FAKE_STATIC_ARCHITECTURES)
+
+
+class FakeIQMClient:
+    """Fake IQM client with class-level fixtures."""
+
+    aliases = FAKE_IQM_ALIASES
+    static_architectures = FAKE_STATIC_ARCHITECTURES
+    dynamic_architectures = FAKE_DYNAMIC_ARCHITECTURES
+    jobs: dict[uuid.UUID, FakeCircuitJob] = {}
```

Ruff (0.15.7)

[warning] Mutable default value for class attribute

(RUF012)